packages/main/src/components/ObjectPage/index.tsx

```diff
@@ -616,18 +613,18 @@ const ObjectPage = forwardRef<HTMLDivElement, ObjectPagePropTypes>((props, ref)
 
   const renderTitleSection = useCallback(
     (inHeader = false) => {
-      const titleStyles = { ...(inHeader ? { padding: 0 } : {}), ...(headerTitle?.props?.style ?? {}) };
+      const titleInHeaderClass = inHeader ? classes.titleInHeader : undefined;
```

Comment:
What about the given `className` on the header? shouldn't we add it as well?

Reply:
Yup, forgot about that. Thanks :)